test/src/org/labkey/test/tests/targetedms/TargetedMSProteinGroupingTest.java

```diff
@@ -0,0 +1,60 @@
+package org.labkey.test.tests.targetedms;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.components.CustomizeView;
+import org.labkey.test.util.DataRegionTable;
+
+@Category({})
```

Comment:
need to specify category here?

Reply:
Last week we decided to make the category blank for all targetedMS tests and control the category via teamcity composite build.